make/protogen.mk

```diff
@@ -24,7 +26,7 @@ GENERATED_API_SWAGGER_SPECS = $(API_SERVICE_PROTOS:%.proto=$(GENERATED_BASE_PATH
 ## Protobuf ##
 ##############
 # Set some platform variables for protoc.
-PROTOC_VERSION := 3.6.1
+PROTOC_VERSION := 21.4
```

Comment:
coincidentally, I was also going to bump this version up https://github.com/stackrox/scanner/pull/867